### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by edgarsperling** [LeonardSchwenk/public-gruppenadressen-generator#43] [Ideas]
Hallo,
anbei ein paar Vorschläge von einem neuen User.
- die Möglichkeit die Reihenfolge der Objekte automatisch zu sortieren
  (So das die Reihenfolge in jedem Raum gleich ist)
- Die Möglichkeit einen Status zu den Geschossen einzutragen
- Die Möglichkeit einen Status zu den Räumen einzutragen
- Status im Projekt/Geschoss/Raum auf der Projektseite anzeigen
- Gruppenadressen Importieren
- Duplizieren von Räumen
- Das Detailfenster geht immer auf der rechten Seite auf, das finde ich für mich sehr gewöhnungsbedürftig.
   Ich fände es auf der linken Seite besser platziert

Ich finde es schon sehr gut umgesetzt.
Ich hoffe das Projekt wird sich gut weiterentwickeln.

**Comment by LeonardSchwenk** [comment]
Hi @edgarsperling,
Danke für dein Feedback und deine Beiträge!

Sehr coole Vorschläge, insbesondere über das importieren hab ich noch nie nachgedacht. Top


Derzeit sind wir mit der Entwicklung eines umfangreichen Updates beschäftigt, das die GA-Strukturen flexibler gestalten soll. 

Sobald dies abgeschlossen ist, sehe ich mehrere Punkte in deinen Anregungen, die sich gut umsetzen lassen.
